docs/loading/BrokerLoad.md

```diff
@@ -82,9 +82,13 @@ Note that in StarRocks some literals are used as reserved keywords by the SQL la
    )
    ENGINE=OLAP
    PRIMARY KEY(`id`)
-   DISTRIBUTED BY HASH(`id`) BUCKETS 10;
+   DISTRIBUTED BY HASH(`id`);
    ```
 
+    > **NOTICE**
+    >
+    > Since v2.5.7, StarRocks can set the number of buckets (BUCKETS) automatically when a table is created or a partition is added. You no longer need to manually set the number of buckets. For detailed information, see [determine the number of buckets](../table_design/Data_distribution.md#determine-the-number-of-buckets).
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/ce44d9a6-c9e7-469f-b416-e2afd29e167b)
See previous comments. There are similar sentences below. Make sure you see to all of them.